src/main/steps/home/get.test.ts

```diff
@@ -104,6 +104,7 @@ describe('HomeGetController', () => {
           divorceOrDissolution: DivorceOrDissolution.DIVORCE,
           state: State.AwaitingApplicant2Response,
           ...jointApplicant2CompleteCase,
+          applicant2WhoIsFinancialOrderFor: [],
```

Comment:
as these are in the nightly prs, might be best to remove it here due to merge conflicts.